.github/workflows/comment.yaml

```diff
@@ -3,14 +3,87 @@ name: Comment on the pull request
 on:
   workflow_call:
 
+env:
+  ARTIFACT_NAME: report
+
 jobs:
   comment-on-pr:
     name: Comment on PR
     runs-on: ubuntu-22.04
     if: github.event.workflow_run.event == 'pull_request'
     steps:
       - uses: actions/checkout@v3
-      - uses: canonical/operator-workflows/.github/actions/comment_pr@main
-        with:
-          artifact-name: report
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Download test report artifact
+      uses: actions/github-script@v6.2.0
```

Comment:
is the indentation correct here? It looks like `uses` is not aligned with `name`

Reply:
Indeed. Thx